doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -642,6 +642,31 @@ The `mz_sessions` table contains a row for each active session in the system.
 | `client_ip`      | [`text`]                       | The IP address of the client that initiated the session.                                                                  |
 | `connected_at`   | [`timestamp with time zone`]   | The time at which the session connected to the system.                                                                    |
 
+
+## `mz_network_policies`
+The `mz_network_policies` table contains a row for each network policy.
+
+<!-- RELATION_SPEC mz_internal.mz_network_policies -->
+| Field            | Type                  | Meaning                                                                                                         |
+| -----------------| ----------------------| --------                                                                                                        |
+| `id`             | [`text`]              | The ID of the network policy.                                                                                   |
+| `name`           | [`text`]              | The name of the network policy.                                                                                 |
+| `owner_id`       | [`text`]              | The role ID of the owner of the network policy. Corresponds to [`mz_catalog.mz_roles`](../mz_catalog#mz_roles). |
```

Comment:
nit (feel free to ignore): Should it be [\`mz_catalog.mz_roles.id\`] or even "the `id` field in [`mz_catalog.mz_roles`]" ?